pytorch_lightning/callbacks/quantization.py

```diff
@@ -175,6 +200,21 @@ def _check_feasible_fuse(self, model):
                 )
         return True
 
+    def _collect_observer_enabled(self):
```

Comment:
typing ?

Reply:
The return type of `_collect_observer_enabled` is added. (`Dict[FakeQuantizeBase, Tensor]`)